services/llm_service.py

```
@@ -2,60 +2,62 @@
     DEPRECATED_MODELS = {
-        'gemini-pro-vision': 'gemini-1.5-flash',
-        'gemini-1.0-pro-vision': 'gemini-1.5-flash',
-        'gemini-1.0-pro-vision-latest': 'gemini-1.5-flash-latest',
-        'gemini-pro': 'gemini-1.5-flash',  # gemini-proも新しいモデルに置き換え
-        'gemini-1.0-pro': 'gemini-1.5-flash'
+        "gemini-pro-vision": "gemini-1.5-flash",
+        "gemini-1.0-pro-vision": "gemini-1.5-flash",
+        "gemini-1.0-pro-vision-latest": "gemini-1.5-flash-latest",
+        "gemini-pro": "gemini-1.5-flash",  # gemini-proも新しいモデルに置き換え
+        "gemini-1.0-pro": "gemini-1.5-flash",
     }
```

Ruff (0.14.6)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)